rotkehlchen/api/v1/resources.py

```diff
@@ -2317,6 +2323,38 @@ def patch(self, asset: AssetWithOracles) -> Response:
         return self.rest_api.refresh_asset_icon(asset=asset)
 
 
+class LocationAssetMappingsResource(BaseMethodView):
+    post_schema = LocationAssetMappingsPostSchema()
+    put_schema = LocationAssetMappingsUpdateSchema()
+    patch_schema = LocationAssetMappingsUpdateSchema()
```

Comment:
no need to duplicate

```suggestion
    update_schema = LocationAssetMappingsUpdateSchema()
```